Sources/AblyAssetTrackingSubscriber/SubscriberDelegate.swift

```diff
@@ -20,6 +20,15 @@ public protocol SubscriberDelegate: AnyObject {
         - location: Received location.
      */
     func subscriber(sender: Subscriber, didUpdateEnhancedLocation location: CLLocation)
+    
+    /**
+     Called when the `Subscriber` receive any Raw Location (matched to road) update for observed trackable
```

Comment:
I don't think that the raw locations are "matched to road" :wink: I'd remove it from this documentation
```suggestion
     Called when the `Subscriber` receive any Raw Location update for observed trackable
```